src/pages/availability-panel/index.tsx

```diff
@@ -9,14 +9,19 @@ import updateTasksStatus from '@/helperFunctions/updateTasksStatus';
 import { AVAILABLE } from '@/constants/task-status';
 import { FEATURE } from '@/constants/task-type';
 import { BASE_URL } from '@/constants/url';
+import { useGetIdleMembersQuery } from '@/app/services/membersApi';
 
 const AvailabilityPanel: FC = () => {
-    const [idleMembersList, setIdleMembersList] = useState<string[]>([]);
     const [unAssignedTasks, setUnAssignedTasks] = useState<task[]>([]);
     const [error, setError] = useState<boolean>(false);
     const [isTaskLoading, setIsTaskLoading] = useState<boolean>(true);
-    const [isMemberLoading, setIsMemberLoading] = useState<boolean>(true);
     const [refreshData, setRefreshData] = useState<boolean>(false);
+    const {
+        data: idleMembersList = [],
+        isError: membersError,
+        isLoading: isMemberLoading,
+        refetch: refreshMemberList,
+    } = useGetIdleMembersQuery();
```

Comment:
if you are not using isLoading  or reFetch then there is no point to write it here

Reply:
Both are used. `isMemberLoading ` was previously used as a state variable so its not reflected in this PR and `refreshMemberList ` is used in the `getData` method.

Reply:
please read about what refetch does in rtk query then use it

Reply:
From what I understand it makes a new request to fetch the current data from the server. Which I believe is same as what was happening in the previous fetch call.